### ISSUE TITLE
contacts keep disappearing

### ISSUE THREAD

**Issue by nathgit** [NullVoxPopuli/emberclear#1661]
I created & logged into 2 different accounts, on 2 different devices just to test this. They both stay logged in but the contact that I added on each one keeps disappearing. Sometimes only on 1, sometimes on the other, sometimes on both. Does the pushpin icon have anything to do with it?

**Comment by NullVoxPopuli** [comment]
Hello! ✨

Is your contact offline? And is 'hide offline contacts' enabled in your settings?

The pushpin is supposed to keep them visible and at the top of your list. 🤔

**Comment by nathgit** [comment]
No. I'm using my phone & chromebook to message back & forth just to try it.
hide contacts was never enabled. 
Should the pushpin be horizontal or vertical?
It seems to happen on page refresh a lot. Maybe that's the issue?

**Comment by NullVoxPopuli** [comment]
If hide contacts was never enabled, the pin doesn't do anything with only one contact. Vertical or horizontal is the same.

What phone browser are you using? 
So, when you refresh the page on either device, you lose all your contacts?

**Comment by nathgit** [comment]
I'm using chrome on both devices.
It seems to happen randomly. Sometimes it will work on refresh, then sometimes even when it's showing both I'll try to send a message & get the error message.

**Comment by nathgit** [comment]
I mean the "delete | resend | resend automatically". When that happens on one it happens on the other.

**Comment by nathgit** [comment]
I've been working with it & it seems to be working now, but we'll see. I'm not sure what I did differently if anything. The qr code doesn't seem to want to scan. It just sits there, but using the invite link works.

**Comment by NullVoxPopuli** [comment]
Strange! I haven't done anything either. 

The qr code scanning might not be error-correcting enough (weird angles or lighting can change what's read). 
I'll take a look at that.

Might not get to this until the weekend tho 🙃 
If you're a developer, any help is always appreciated <3

**Comment by nathgit** [comment]
I'm not a developer, but I'll be happy to keep testing it & report bugs if I find any. I also have a mastodon & mewe account so I can mention it there. It's always nice to find a responsive developer. I can think of a few that weren't. Should I leave this open or close it?

**Comment by NullVoxPopuli** [comment]
Open would be great. I'll have more info by the end of the weekend 🔥
